internal/core/components/rewriter.go

```diff
@@ -90,12 +91,7 @@ func (r *Rewriter) write(key string, hubKey storer.HubKey, nodes []*entity.Node,
 	_ = storer.GetStore(hubKey).UpdateNodes(r.ctx, key, nodes)
 }
 
-// Divide the different types of entities
-func divideEntities(entities []string) map[string][]string {
-	divides := make(map[string][]string)
-	for _, entityID := range entities {
-		entityTyp, id := discoverer.DecodeEntityID(entityID)
-		divides[entityTyp] = append(divides[entityTyp], id)
-	}
-	return divides
+func getHubKey(entityID string) storer.HubKey {
+	s := strings.Split(entityID, "/")[2]
```

Comment:
We can't assume the id is "/apisix/routes/1", as the prefix is configurable. We should use `$prefix/routes/$name`